--- annobin.orig/gcc-plugin/annobin.cc	2023-04-28 09:58:05.487156290 +0100
+++ annobin-12.09/gcc-plugin/annobin.cc	2023-04-28 09:58:36.434248542 +0100
@@ -2104,7 +2104,7 @@ annobin_emit_start_sym_and_version_note
 	     Ensure that we do not have empty special text sections so that the
 	     annobin start symbols are never beyond the end of the sections.  */
 #ifndef ANNOBIN_NOP_INSN
-#define ANNOBIN_NOP_INSN ".nop"
+#define ANNOBIN_NOP_INSN "nop"
 #endif
 	  if (* suffix && enable_ppc64_nops)
 	    annobin_emit_asm (ANNOBIN_NOP_INSN,
